namer/consul/src/main/scala/io/buoyant/namer/consul/ConsulInitializer.scala

```diff
@@ -52,7 +80,8 @@ case class ConsulConfig(
   consistencyMode: Option[ConsistencyMode] = None,
   failFast: Option[Boolean] = None,
   preferServiceAddress: Option[Boolean] = None,
-  weights: Option[Seq[TagWeight]] = None
+  weights: Option[Seq[TagWeight]] = None,
+  tls: Option[TlsConsulConfig] = None
```

Comment:
I think you can use io.buoyant.linkerd.TlsClientConfig here instead of defining a new TlsConsulConfig class.

Reply:
I have thought about it. I can use io.buoyant.linkerd.TlsClientConfig if you prefer.